pkg/controller/dataimportcron-controller.go

```diff
@@ -822,11 +822,13 @@ func (r *DataImportCronReconciler) garbageCollectPVCs(ctx context.Context, names
 		})
 		for _, pvc := range pvcList.Items[maxImports:] {
 			dv := cdiv1.DataVolume{ObjectMeta: metav1.ObjectMeta{Name: pvc.Name, Namespace: pvc.Namespace}}
+			r.log.V(3).Info("Calling delete on dv", "dv.Name", dv.Name)
```

Comment:
Maybe wait for https://github.com/kubevirt/containerized-data-importer/pull/2754 to be merged since this code will be slightly modified. Other than that looks good.

Reply:
that makes sense to me, maybe close this PR and push those few lines in there?
@arnongilboa

Reply:
added, please check if it lgty. Dropped the annotation.

Reply:
I am keeping this PR until we clear up which version GC gets in to

Reply:
@akalenyu  #2754 merged, including the logs.